src/thor/unidirectional_astar.cc

```diff
@@ -518,6 +518,17 @@ std::vector<std::vector<PathInfo>> UnidirectionalAStar<expansion_direction, FORW
       edgestatus_.Update(pred.edgeid(), EdgeSet::kPermanent);
     }
 
+    // setting this edge as settled
```

Comment:
we also want the other statuses, i.e. reached & connected, see https://valhalla.github.io/valhalla/api/expansion/api-reference/ and/or bidir a*